brokerapi/brokers/bigquery/broker.go

```diff
@@ -59,7 +60,7 @@ func (b *BigQueryBroker) Provision(instanceId string, details models.ProvisionDe
 		params["name"] = name_generator.Basic.InstanceName()
 	}
 
-	service, err := googlebigquery.New(b.Client)
+	service, err := googlebigquery.New(b.HttpConfig.Client(context.Background()))
```

Comment:
did these services not have the `models.CustomUserAgent` before? I had assumed that got set on the shared client object, but I also had assumed that was used by all the APIs.